verilog/tools/ls/vscode/src/download-ls.ts

```diff
@@ -29,6 +36,16 @@ export async function checkAndDownloadBinaries(
   binaryPath: string,
   output: vscode.OutputChannel
 ): Promise<string> {
+
+  output.appendLine("Platform: '" + platform() + "'");
+
+  // Update home paths to an absolute path
+  if(platform() != "win32" && binaryPath.startsWith("~/")) {
```

Comment:
To be consistent with the rest of the file, leave a space after the `if`.

Reply:
Done